frontend/src/services/jobSeekersService.js

```diff
@@ -0,0 +1,35 @@
+import axios from "axios"
```

Comment:
This file looks identical to `frontend/services/jobSeekersService.js`. Is there a need to keep both?